core/src/main/java/org/elasticsearch/search/aggregations/support/ValuesSourceConfig.java

```diff
@@ -76,6 +76,21 @@ public boolean valid() {
         return this;
     }
 
+    public ValuesSourceConfig<VS> format(final DocValueFormat format) {
```

Comment:
I went ahead and did it in this PR but will keep it as a separate commit when I merge.